### ISSUE TITLE
Will UO Legacy ever be updated again?

### ISSUE THREAD

**Issue by antdude** [gorhill/uBlock-for-firefox-legacy#348]
It hasn't been updated since 6/20/2021. I still use old web browsers like SeaMonkey. :(

**Comment by Gittyperson** [comment]
It does its job fine currently, right? As long as filterlists work, there shouldn't be any pressing need for uBO updates. Only time will tell.

Are there any important filterlists that do not work with the current legacy version and require a uBO update?

**Comment by antdude** [comment]
Well, YouTube started blocking ad blockers like UO. It doesn't seem like there's a way to block ads in the legacy version. :(

**Comment by Gittyperson** [comment]
Haven't noticed this. Videos play fine, as before.

**Comment by antdude** [comment]
Are you logged in with an account? Also, it's fine with vertical shorts. Recent (e.g., today's) long videos show this problem.

**Comment by Gittyperson** [comment]
Yes, logged in. I don't see a Recent section. Are you sure your filterlists aren't messed up?

**Comment by antdude** [comment]
I meant recently uploaded videos like in https://www.youtube.com/feed/subscriptions when logged in. I use the subscribed chrome://ublock0/content/dashboard.html#3p-filters.html filter lists. I even told my UO ext. to reset back to default settings and still see the same blocked YouTube videos. 

Like with yesterday's https://www.youtube.com/watch?v=AWhJ4LuVcRU video, I see:
"Ad blockers violate YouTube's Terms of Service

    It looks like you may be using an ad blocker. Video playback is blocked unless YouTube is allowlisted or the ad blocker is disabled.
    Ads allow YouTube to be used by billions worldwide.
    You can go ad-free with YouTube Premium, and creators can still get paid from your subscription.

Try YouTube Premium
Not using an ad blocker? Report issue" https://www.reddit.com/r/youtube/comments/173iyie/bye_bye_youtube/ has a screen shot of it.

I don't know if it matters, but I am in USA. Maybe a region thing? However, I see http://reddit.com/r/youtube mention other countries are seeing it too. :(

**Comment by Gittyperson** [comment]
I don't see any of that. Loki video plays fine. Also tried a few recent videos from subscribed channels.

It could be a US thing, I'm in the EU. Try with a VPN. If those Reddit reports are correct, then this is not related to uBO Legacy specifically, as you imply.

Some of the uBO Legacy filterlists could be broken due to invalid URL. Online Malicious URL Blocklist is an example. You should disable it, locate the correct URL (e.g. from filterlists.com) and add it as a Custom list.

**Comment by antdude** [comment]
Interesting. Disabling Online Malicious URL Blocklist didn't help on my end. What other uBO Legacy filterlists are invalid/broken? uBO ext. needs to be updated to remove them and add new ones from its default filter list. ;)  I don't have an outside country VPN.  Also, which web browser(s) are you using? SeaMonkey v2.53.17.1 (User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Firefox/91.0 SeaMonkey/2.53.17.1) for me.

**Comment by Gittyperson** [comment]
The rest of the filterlists should be ok. Haven't tested them all though. Actually, there is a newer uBO Legacy version you can try, even though I doubt it will solve your issue. I assume you've already tried clearing all cache/history/cookies etc.

https://forum.palemoon.org/viewtopic.php?f=46&t=28149

**Comment by garoto** [comment]
I only subscribe to the native uBlock lists, and block third-party {scripts|frames} globally by default, whitelist as necessary. Large, generic filter lists are a major source of nuisance.

**Comment by pmpkn72** [comment]
I can confirm antdude's report re. YouTube's player getting blocked only when using the legacy version of uBO.

Updating to the latest filters on PaleMoon (my usual browser) and Chrome has totally different effects, since Chrome can run uBO v1.52 while PM can't. Here's the reddit thread explaining how uBO is currently bypassing YT's anti-adblocker scripts:

https://old.reddit.com/r/uBlockOrigin/comments/17j6ygs/youtube_antiadblock_and_ads_october_29_2023_mega/

Clearing cookies etc makes no difference in my case.

**Comment by Gittyperson** [comment]
Have you subscribed to uBO's "Quick fixes" filterlist? That's included in the default lists for uBO, but not in the Legacy version. You can -and should- import it into your Custom lists. No issues with YT here.

Make good use and selection of filterlists. "I only subscribe to the native uBlock lists" does not cut it, otherwise there wouldn't be a need for more filterilsts. Even more so for the old Legacy version which works just fine, but hasn't had its default lists selection updated in a while, obviously.

**Comment by antdude** [comment]
Newbie question: How the heck do I/we add this filter list? I see moz-extension://964e6ee0-f644-4277-a857-6fe026fc6486/asset-viewer.html?url=ublock-quick-fixes. :(

Since UO legacy uses old filter list, what subscriptions should we use?

**Comment by Gittyperson** [comment]
That doesn't look like a proper URL, but something Firefox-related. I assume you are not using Firefox. Here are two solutions for you.

1. Look for the proper URL in filterlists.com, copy it, go to uBO Legacy/Filter lists/Custom/Import/Paste/Apply changes
2. Copy the proper URL below and do the above.

`https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/quick-fixes.txt#`

**Comment by antdude** [comment]
I was using Firefox v119's UO ext. and its external https://github.com/uBlockOrigin/uAssets to find that Quick Fixes subscription's URL so I could add into my old school SeaMonkey (SM)'s UO legacy ext. I added it into the bottom of my chrome://ublock0/content/dashboard.html#3p-filters.html in SM. I hope it works!

**Comment by antdude** [comment]
Hmm, it doesn't seem to help at all when logged in. I even logged out and relogged into my YouTube/Google account. :(

**Comment by pmpkn72** [comment]
> Have you subscribed to uBO's "Quick fixes" filterlist? That's included in the default lists for uBO, but not in the Legacy version. You can -and should- import it into your Custom lists. No issues with YT here.
> 
> Make good use and selection of filterlists. "I only subscribe to the native uBlock lists" does not cut it, otherwise there wouldn't be a need for more filterilsts. Even more so for the old Legacy version which works just fine, but hasn't had its default lists selection updated in a while, obviously.

Yes, I've been subscribed to that filterlist for a while.

As for your lack of issues with YT, do you get any of the YT warnings against adblokers if you disable uBO or that paticular filterlist? Because if you don't, you simply haven't been targeted yet. uBO would have nothing to do with it in that case.

**Comment by Movisivi** [comment]
HahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahahaHahahahahahahahahaha
